### ISSUE TITLE
Feature Request: PayPal Subscriptions

### ISSUE THREAD

**Issue by bays-media** [concrete5-community-store/community_store_paypal_standard#7]
Am I right in thinking that PayPal subscriptions (allow you to setup automatic, recurring payments) is not a function of this? If so, will this functionality be integrated in the future?

**Comment by Mesuva** [comment]
Yes, it's not something supported, it only makes normal immediate transactions.

It's not on the radar to support subscriptions, the store doesn't have fields against products to configure things like the billing cycles (to name one of the simpler aspects not handled, there are quite a number of more complex aspects that would have to be considered or added)

**Comment by bays-media** [comment]
No problem I get the complexities here. Thanks for the swift response.

**Comment by pinecreativelabs** [comment]
You can still have paid memberships, even without using Paypal subscriptions. Here's how it could be done: 

1. In Concrete5 Members area, set up a group and set it so that the user is automatically removed from the group after a certain amount of time.
2. Using Community Store, while adding a product, use the Memberships tab to create user account, and here you can add them to the expiring group you created. 
 
Granted, that still doesn't solve the automated recurring payments issue, but at least you'll be able to limit their access.
